docs/data/stashcache/install-cache.md

```diff
@@ -5,192 +5,222 @@ This document describes how to install a StashCache cache service.  This service
+and the administrative and security contacts.
+
+
+### Initial registration
+
+To register your cache host, follow the general registration instructions [here](https://opensciencegrid.org/docs/common/registration/#new-resources).
+The service type is `XRootD cache server`.
+
+!!! info
+    This step must be completed before installation.
+
+In your registration, you must specify which VOs your cache will serve by adding an `AllowedVOs` list, with each line specifying a VO whose data you are willing to cache.
+
+There are special values you may use in `AllowedVOs`:
+
+- `ANY_PUBLIC` indicates that the cache is willing to serve public data from any VO.
+- `ANY` indicates that the cache is willing to serve data from any VO,
+  both public and non-public.
+  `ANY` implies `ANY_PUBLIC`.
+
+There are extra requirements for serving non-public data:
+
+- In addition to the cache allowing a VO in the `AllowedVOs` list,
+  that VO must also allow the cache in its `AllowedCaches` list.
+  See the page on [getting your VO's data into StashCache](/data/stashcache/vo-data).
+- There must be an authenticated XRootD instance on the cache server.
+- There must be a `DN` attribute in the resource registration
+  with the [subject DN](/security/host-certs#before-starting) of the host certificate
+
+This is an example registration for a cache server that serves all public data:
+```yaml
+  MY_STASHCACHE_CACHE:
+    FQDN: my-cache.example.net
+    Service: XRootD cache server
+      Description: StashCache cache server
+    AllowedVOs:
+      - ANY_PUBLIC
+```
+
+This is an example registration for a cache server that only serves authenticated data from the OSG VO:
+```yaml
+  MY_AUTH_STASHCACHE_CACHE:
+    FQDN: my-auth-cache.example.net
+    Service: XRootD cache server
+      Description: StashCache cache server
+    AllowedVOs:
+      - OSG
+    DN: /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=my-auth-cache.example.net
+```
+
+
+### Finalizing registration
+
+Once initial registration is complete, you may start the installation process.
+In the meantime, open a [help ticket](https://support.opensciencegrid.org) with your cache name.
+Mention in your ticket that you would like to "Finalize the cache registration."
+
+
 Installing the Cache
 --------------------
 
-The StashCache daemon consists of an XRootD server and a service for collecting and reporting
-statistics about the cache. To simplify installation, OSG provides convenience RPMs that install all required
-software with a single command:
+The StashCache software consists of an XRootD server with special configuration and supporting services.
+To simplify installation, OSG provides convenience RPMs that install all required
+packages with a single command:
 
-    :::console
-    root@host # yum install --enablerepo=osg-testing stashcache-cache-server
+```console
+root@host # yum install --enablerepo=osg-testing stash-cache
```

Comment:
```suggestion
root@host # yum install stash-cache
```

Reply:
I would like to leave it in until it's released. It would be useful to be able to push this branch as-is to the ITB docs site so folks like Marian or Igor who are testing this can use it. We can quickly make a separate PR to get rid of all the --enablerepo args once we finally manage to release xcache.

Reply:
Done.